packages/concatjs/_README.md

```diff
@@ -117,19 +117,3 @@ browser_repositories(
     firefox = True,
 )
 ```
-
-## Installing with self-managed dependencies
```

Comment:
isn't there some new version of this guidance? are we trying to deprecate "self-managed dependencies"?

Do they just need to move //:node_modules to data?

Reply:
Restored with updated instructions